game/src/common/speed.rs

```diff
@@ -171,17 +166,16 @@ impl SpeedControls {
                     let now = Instant::now();
                     self.state = State::Running {
                         last_step: now,
-                        speed_description: "...".to_string(),
                         last_measurement: now,
                         last_measurement_sim: current_sim_time,
                     };
+                    self.slider.set_percent(ctx, current_sim_time.inner_seconds()/86400.0);
```

Comment:
Did current_sim_time / Duration::END_OF_DAY not work?

Reply:
Doesn't work. current_sim_time is f64, and Duration::END_OF_DAY is not f64.
A question regarding to data type, why all the dimensions used are f64? width/height are all integers.

Reply:
I tried `self.slider.set_percent(ctx, current_sim_time / Duration::END_OF_DAY);` and it seemed to work.
I don't have a consistent story for units yet. Maybe screen dimensions and more should be integers; I need to audit all of the geometry and GUI types and rethink some of them.